crates/wasm/src/tx.rs

```diff
@@ -103,34 +82,174 @@ pub fn build_tx(
     return serde_wasm_bindgen::to_value(&tx).unwrap();
 }
 
+/// Get transaction view, transaction perspective
+/// Arguments:
+///     full_viewing_key: `bech32 String`
+///     tx: `pbt::Transaction`
+/// Returns: `TxInfoResponse`
+#[wasm_bindgen]
+pub async fn transaction_info(full_viewing_key: &str, tx: JsValue) -> Result<JsValue, Error> {
+    let fvk = string_to_fvk(full_viewing_key)?;
+
+    let transaction = serde_wasm_bindgen::from_value(tx)?;
+    let response = transaction_info_inner(fvk, transaction).await?;
+
+    serde_wasm_bindgen::to_value(&response)
+}
+
+/// deprecated
```

Comment:
If deprecated, can we delete? This applies to all the functions labelled with `/// deprecated`

Reply:
That means it still uses [penumbra-zone/wallet](https://github.com/penumbra-zone/wallet) , so we can't remove it yet.

Reply:
✅